src/classes/COFF_Affiliation_TDTM.cls

```diff
@@ -471,20 +489,20 @@ public class COFF_Affiliation_TDTM extends TDTM_Runnable {
     }
 
    /*******************************************************************************************************
-    * @description Determines whether an Affiliation to the Department the Course Offering belongs to already
-    * exists for each faculty member. Public only so it can be called from the tests.
+    * @description Determines whether an Affiliation to the Account the Course Offering belongs to already
+    * exists for each faculty member. Will return null if no Affiliation exists.
+    * @return Affiliation__c Affiliation to the Account or null
     ********************************************************************************************************/
-    public Boolean afflToDptExists(Course_Offering__c offering, Map<ID, List<Affiliation__c>> facultyIDtoAffls) {
+    @testVisible
+    private Affiliation__c affiliationToAccount(Course_Offering__c offering, Map<ID, List<Affiliation__c>> facultyIDtoAffls) {
         List<Affiliation__c> facultyMemberAffls = facultyIDtoAffls.get(offering.Faculty__c);
```

Comment:
It's not clear to me why change the return value from Boolean to Affiliation__c. It doesn't seem like anything is done with the returned affiliation in any of the places this method is called.